docs/standards.md

```diff
@@ -0,0 +1,167 @@
+
+##### Meta
+Add a description of the "meta" patterns here. This section still needs to be added to SG2.
+
+### How to classify/name patterns
+All pattern class names should follow standard [BEM practices](http://getbem.com/naming/) and be prefixed according to their theme (e.g. `ama__`). 
+
+Below is an example of how a breadcrumb organism for the default AMA theme may appear:
+```html
+<section class="ama__breadcrumbs ">
+  <nav class="ama__breadcrumbs__container">
+    <ol class="ama__breadcrumbs__items">
+      <li class="ama__breadcrumbs__item"><a href="#">Link 1</a></li>
+      <li class="ama__breadcrumbs__item"><a href="#">Link 2</a></li>
+    </ol>
+  </nav>
+</section>
+```
+This pattern shows that `breadcrumbs` is a block of the `ama` theme and has the elements `container`, `items` and `item`.
+
+#### Pattern classification
+See "PatternLab Organization" above. 
+@todo - figure out a good/bad example.
+
+#### Pattern scalability
+It's important for patterns to be easily reusable.
+- Patterns should use JSON data from an associated `.json` file for their content:
+  - Bad: `<a href="link.html">I am a link</a>`
+  - Good: `<a href="{{ link.url }}">{{ link.text }}</a>`
+
+- Patterns should be named something that won't exclude them from being used somewhere else in the future.*
+  - Bad: an atom named `login-form-password-field.twig`
+  - Good: an atom named `password-field.twig`
+  
+  \* unless they are **not** intended to be reusable. This rule may apply more to smaller patterns (i.e. atoms). 
+
+- Patterns should be coded to accept CSS classes or other HTML attributes from the patterns that include them.
+  - Bad: `<div class="foo">{{ div.content }}</div>`
+  - Good: `<div class="foo {{ div.class }}">{{ div.content }}</div>`
+
+- Patterns should not include hard-coded styles or classes that may not apply to all instances in which the pattern is used.
+  - Bad: `<div class="dropdown__list-item dropdown__list-item-foo dropdown__list-item-bar"></div>`
+  - Good: `<div class="dropdown__list-item {{class}}"></div>`
+  
+#### Pattern consistency
+- Patterns should generally adhere to the existing name/organization schema. File names, pattern organization within PatternLab, and code should appear in kind with the existing patterns in the library.
+
+   - Bad: `<DIV><A HREF="{{ url }}"></A></ BR>`
+  - Good: `<div><a href="{{ url }}"</a><br>`
+
+#### Pattern namespacing
+Pattern code should comply with the BEM naming system. 
+
+  - Bad: `differentlyCapitalized.twig`  
+  - Bad: `named-with-a-number2.twig`
+  - Bad: `named_with_underscores.twig`
+  - Bad: `atoms/our-special-project/header.twig`
+  - Bad:
+  ```twig
+    <div class="header">
+      <a href="{{ url }}" class="fancy-link">{{ content }}</a>
+      <img src="{{ src }} class="fun-image">
+    </div>
+  ```
+   - Good:
+   ```twig
+    <div class="header">
+      <a href="{{ url }}" class="header__link-fancy">{{ content }}</a>
+      <img src="{{ src }} class="header__image-fun">
```

Comment:
`header__link-fancy` and `header__image-fun` should both be using double hyphens for the modifiers; meaning that they should be `header__link--fancy` and `header__image--fun`.

Reply:
![bitmoji](https://render.bitstrips.com/v2/cpanel/bea8cce1-c2d2-4631-8975-65c26599d82d-b9e899d3-9eb0-4b4d-a8ff-dc5da32696f1-v1.png?transparent=1&palette=1&width=246)